applications/nrf_desktop/src/modules/Kconfig.config_channel

```diff
@@ -78,6 +78,38 @@ config DESKTOP_CONFIG_CHANNEL_DFU_SYNC_BUFFER_SIZE
 	  The host must perform progress synchronization at least
 	  every synchronization buffer bytes count.
 
+config DESKTOP_CONFIG_CHANNEL_DFU_GENERATION
```

Comment:
BTW, are we planning to support the generation string for the MCUmgr DFU transport?

Reply:
Currently there is no plan to do it (MCUmgr DFU transport is not supported in the LVFS too)